sql/sql_table.cc

```diff
@@ -9611,17 +9612,25 @@ enum fk_column_change_type
                                  change in foreign key column.
   @retval FK_COLUMN_RENAMED      Foreign key column is renamed.
   @retval FK_COLUMN_DROPPED      Foreign key column is dropped.
+  @retval FK_COLUMN_NOT_NULL     Foreign key column cannot be null.
```

Comment:
The comment is a little confusing. Should it be: "ON...SET NULL or ON UPDATE CASCADE conflicts with NOT NULL"?

I see that there are some tests cases that cover the following type of scenario:
```sql
create table t1 (f1 int null unique) engine=innodb;
create table t2 (f1 int primary key,
    constraint c1 foreign key (f1) references t1(f1)
    on update cascade) engine=innodb;
insert into t1 set f1=0;
insert into t2 set f1=0;
update t1 set f1=null;
```
Here, if we did not refuse to create such a child table, the `update` should lead to a transaction rollback.